src/components/plots/barPlot.ts

```diff
@@ -489,6 +489,33 @@ export module Plot {
       };
     }
     //===== /Hover logic =====
+
+    protected _getAllPlotData(datasetKeys: string[]): PlotData {
+      var plotData = super._getAllPlotData(datasetKeys);
+
+      var valueScale = this._isVertical ? this._yScale : this._xScale;
```

Comment:
We should really refactor the code base to use keyScale/valueScale semantics or something like that instead of xScale/yScale sometime...